Jenkinsfile

```diff
@@ -488,6 +492,7 @@ pipeline {
             }
           }
           steps {
+            build job: 'tool-turn-on-slaves', parameters: [string(name: 'RoleNameAndRGname', value: 'azure-slave-0@kernel_pipeline')], wait: false
```

Comment:
here is an egg and chicken problem: which slave will have the azure label in order to start another azure labeled slave?